Source/JavaScriptCore/llint/LowLevelInterpreter64.asm

```diff
@@ -3465,6 +3472,11 @@ llintOpWithMetadata(op_enumerator_get_by_val, OpEnumeratorGetByVal, macro (size,
 .getSlowPath:
     callSlowPath(_slow_path_enumerator_get_by_val)
     dispatch()
+
+.osrReturnPoint:
```

Comment:
When we jump to this case? I didn't find the code for jumping to `.osrReturnPoint`.

Reply:
This is used when OSR exit happens for the inlined function calls for this opcode. See returnLocation code.